drift/test/generated/todos.dart

```diff
@@ -159,7 +157,7 @@ class CustomRowClass {
 
 class PureDefaults extends Table {
   // name after keyword to ensure it's escaped properly
-  TextColumn get txt =>
```

Comment:
We also have the example and so on, but IMO we should leave a few of these unchanged to make sure the existing syntax isn't negatively affected.

Reply:
Done